tracker/views.py

```diff
@@ -15,10 +14,62 @@
 
 
 from tracker.models import Student, Standard, Assessment
-
-
-def hello_world(request):
-    return HttpResponse("Hello, world!")
+from django.shortcuts import redirect
+from django.contrib.auth import authenticate, login
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import logout
+from django.contrib.auth.forms import AuthenticationForm
+from django.contrib import messages
+
+
+
+def login(request):
+    if request.method == 'POST':
+        form = AuthenticationForm(request=request, data=request.POST)
+        if form.is_valid():
+            username = form.cleaned_data.get('username')
+            password = form.cleaned_data.get('password')
+            user = authenticate(username=username, password=password)
+            if user is not None:
+                login(request, user)
+                messages.info(request, f"You are now logged in as {username}")
```

Comment:
You want to display the messages, you can use something like this in `base.html`: https://django-bootstrap4.readthedocs.io/en/latest/templatetags.html#bootstrap-messages